olive/auto_optimizer/config_template/pass_capability.yaml

```diff
@@ -0,0 +1,55 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
```

Comment:
Please add a comment describing how this yaml is used and what needs to be done to list a new pass here.

Reply:
Added.